task/topic.go

```diff
@@ -41,7 +42,7 @@ func LoadTopicsMap(ctx context.Context, serviceAuthToken string, topicClient top
 	} else {
 		err := errors.New("root topic private items is nil")
 		log.Error(ctx, "failed to dereference root topic items pointer", err)
-		return topicMap
+		return topicMap, nil
```

Comment:
If these are errors, then they should probably ush to the errChan too. However, if these are just unusual circumstances and should be log-only, then consider changing the logging level to `Info` instead. (Same for 'error' below on line 56 too)